clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -81,136 +84,134 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-endpoint"
-                      textToCopy={storageFields.s3Fields.endpoint}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Region"
-                    isEmpty={!storageFields.s3Fields.region}
-                    contentWhenEmpty="No region"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-region"
-                      textToCopy={storageFields.s3Fields.region || ''}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Bucket"
-                    isEmpty={!storageFields.s3Fields.bucket}
-                    contentWhenEmpty="No bucket"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-bucket"
-                      textToCopy={storageFields.s3Fields.bucket}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Path"
-                    isEmpty={!storageFields.s3Fields.path}
-                    contentWhenEmpty="No path"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-path"
-                      textToCopy={storageFields.s3Fields.path}
-                    />
-                  </DashboardDescriptionListGroup>
-                </>
-              )}
-              {(storageFields?.uri || storageFields?.ociUri) && (
-                <>
-                  <DashboardDescriptionListGroup
-                    title="URI"
-                    isEmpty={!modelArtifact?.uri}
-                    contentWhenEmpty="No URI"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-uri"
-                      textToCopy={modelArtifact?.uri || ''}
-                    />
-                  </DashboardDescriptionListGroup>
-                </>
-              )}
+    <Card>
+      <CardHeader>
+        <Title headingLevel="h2">Version details</Title>
+      </CardHeader>
+      <CardBody>
+        <Flex
+          direction={{ default: 'column', md: 'row' }}
+          columnGap={{ default: 'columnGap4xl' }}
+          rowGap={{ default: 'rowGapLg' }}
+        >
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <DescriptionList isFillColumns>
+              <EditableLabelsDescriptionListGroup
+                labels={getLabels(mv.customProperties)}
+                isArchive={isArchiveVersion}
+                allExistingKeys={Object.keys(mv.customProperties)}
+                title="Labels"
+                contentWhenEmpty="No labels"
+                labelProps={{ variant: 'outline' }}
```

Comment:
As mentioned on Slack, this might be something we want to think about applying to the other places where labels appear - model details and the models/versions tables - either by passing labelProps in those places or even using this outline variant as the default inside EditableLabelsDescriptionListGroup, which is in the mod-arch-shared package.

I don't think this is a PR blocker either, but if others have an opinion we can follow up with another task here.